lib/features/thread/presentation/thread_controller.dart

```diff
@@ -308,6 +312,10 @@ class ThreadController extends BaseController {
     });
   }
 
+  void _initializeIsolateExecutor() async {
+      await _isolateExecutor.warmUp(log: BuildUtils.isDebugMode);
```

Comment:
should `try/catch` because if the isolate is already created it will throw an exception